org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -123,22 +123,27 @@ function bootstrapcivicrm_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
 }
 
 /**
- * Implementation of hook_civicrm_pageRun
+ * Implementation of hook_civicrm_coreResourceList
  */
-function bootstrapcivicrm_civicrm_pageRun($page) {
-  if (!(isset($_GET['snippet']) && $_GET['snippet'] == 'json')) {
-    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
+function bootstrapcivicrm_civicrm_coreResourceList(&$items, $region) {
+  if ($region == 'html-header') {
+    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css', -50, 'html-header');
+    CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/select2-bootstrap.css', -49, 'html-header');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/radio-checkbox.js');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/add-missing-date-addons.js');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/transition.js', 1000, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/scrollspy.js', 1000, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/dropdown.js', 1000, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/collapse.js', 1000, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/modal.js', 1000, 'html-header');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrap', 'js/tab.js', 1000, 'html-header');
   }
 }
 
 /**
  * Implementation of hook_civicrm_buildForm
  */
 function bootstrapcivicrm_civicrm_buildForm($formName) {
-  CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/radio-checkbox.js');
-  CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/add-missing-date-addons.js');
   if($formName == 'CRM_Contact_Form_Search_Advanced') {
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/enable-select2.js');
```

Comment:
You are referring to https://github.com/civicrm/civihr/pull/1105/commits/369d0389088bfad152ccb7b08bb8b47d471f1725 ? To be honest I haven't followed this, so I'm not sure if it's really needed or not. Just to give a bit of context: this is part of our effort to provide a civihr-like look-and-feel to core pages, when I'll have time to review what's been done so far I'll be glad to throw the script away if it's really not needed